src/modules/Dropdown/Dropdown.js

```diff
@@ -8,6 +8,8 @@ import META from '../../utils/Meta'
 import { useKeyOnly, useKeyOrValueAndKey } from '../../utils/propUtils'
 import { iconPropRenderer } from '../../utils/propUtils'
 import keyboardKey from '../../utils/keyboardKey'
+import { makeDebugger } from '../../utils/debug'
+import { objectDiff } from '../../utils/utils'
```

Comment:
New tested util for generating naive object diffs, for development / debugging only.